deployment/scripts/update

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
Add a comment to refer to the development `scripts/update` when making changes to this file and vice versa.